### ISSUE TITLE
Add integer support to TVectorT

### ISSUE THREAD

**Issue by ynikitenko** [root-project/root#8517]
### Is your feature request related to a problem? Please describe.
I'd like to store a vector of integers in a TFile. Rene Brun suggested using [TVectorD for doubles](https://root-forum.cern.ch/t/writing-simple-variable-in-root-files/11094/2?u=ynikitenko), but I'd like to store integers (to store them in the vector of doubles is possible, but less convenient).

[TVectorT](https://root.cern.ch/doc/master/classTVectorT.html) has support only for floats and doubles. [As of 2011, it doesn't support integers](https://root-forum.cern.ch/t/tvectort-int/12041/2?u=ynikitenko).
But I think it would be nice to add that support.

When I try to use TVectorT\<int\> in `ROOT 6.24/00`, `ld` gives an error:

> undefined reference to `TVectorT<int>::TVectorT(int)'

### Describe the solution you'd like
Add support for TVectorT\<int\> and a `TVectorI` shortcut (would be easier to use that from Python).

### Describe alternatives you've considered
@pcanal [proposed](https://root-forum.cern.ch/t/tvectort-int/12041/4?u=ynikitenko) such a code,

    struct VectorWrapper : public TObject {
        std::vector<int> fContent;
        ClassDef(VectorWrapper, 2);
    };

and to generate, compile and link a dictionary for that. However, it would take much more time than to use a ready class.

**Comment by pcanal** [comment]
You can now just use `std::vector<int>` directly:
```
std::vector<int> v;
...
f->WriteObject(&v, "somename");
```
For quite a while now, inheriting from TObject is no longer a requirement for storing an object in a `TFile`

Cheers,
Philippe.

**Comment by ynikitenko** [comment]
Hi Philippe, thanks for the info!

I'll try that. It's a pity that I couldn't find a more recent answer to that question during the search. 
Is there a recent way to write one value to a ROOT file, e.g. a double?

Cheers,
Yaroslav

**Comment by pcanal** [comment]
For numerical type, you would use `TParameter<double>` (i.e. you still need a class for data stored directly in a TFile/TDirectory.  You do not need an object if the data is stored in a TTree).

**Comment by ynikitenko** [comment]
Thanks, Philippe. I added that information to the old thread on the forum. I also found what you were saying in the manual (so it is present somewhere in the documentation, though I usually read the reference guide).
About my initial feature request: I think this is a continuous wish of users to have a vector for ints (and other types) when you have a vector for doubles. So it's up to you whether you add this support or not (me personally no longer need that).
